### ISSUE TITLE
config-template does not add network_name and singleton_availability_zone to tiles without jobs

### ISSUE THREAD

**Issue by kcboyle** [pivotal-cf/om#487]
Observed behavior:

1. run `om config-template` for `p-event-alerts` (as an example
1. upload product to opsman
1. fill out `required-vars.yml`
1. run `om configure-product --config product.yml --vars-file default-vars.yml --vars-file required-vars.yml --vars-file errand-vars.yml --vars-file resource-vars.yml`

Result:
```
could not execute "configure-product": Expected to find variables: network_name
singleton_availability_zone
```

Expectation:
I should be able to run `config-template` for a tile without jobs, fill in required vars, and `configure-product` with `product.yml`+all vars files without issue.

**Comment by kcboyle** [comment]
Through shallow research, we believe this is an issue for all tiles that do not have a vm deployed via a job

**Comment by kcboyle** [comment]
this will be released in om 4.7.0
